### PR TITLE
Update performance.rst

### PR DESCRIPTION
**Pull request by seferov**
Remove deprecated suggestion

See: https://github.com/symfony/symfony/issues/20668

### CODE REVIEW

File: performance.rst [hunk 1]
```diff
@@ -165,8 +165,12 @@ For more details, see :doc:`/components/class_loader/cache_class_loader`.
 .. index::
    single: Performance; Bootstrap files
 
-Use Bootstrap Files
--------------------
+Use Bootstrap Files (For < PHP 7)
```
Comment (wouterj) [inline performance.rst]: I think we can leave the title as it was

File: performance.rst [hunk 2]
```diff
@@ -165,8 +165,12 @@ For more details, see :doc:`/components/class_loader/cache_class_loader`.
...
+---------------------------------
+
+.. note::
+
+    For PHP 7 and higher versions it is not necessary. It is deprecated for Symfony 4.0.
```
Comment (wouterj) [inline performance.rst]: I believe we used `caution` directives for deprecations in the docs. I think we should also provide a little bit of context to this message, like this:

```rst
.. caution::

    The performance optimalizations in PHP 7 removed any overhead. As such,
    using the bootstrap file is discouraged when using PHP 7+ and was
    deprecated in Symfony 4.0. 
```

**Review by wouterj — CHANGES_REQUESTED** [review]
Great find! Thanks for adding this little note to the docs.

I've added 2 comments where I think this PR can be a little bit updated. If you agree and have some free time, please update your PR (by pushing to the `patch-3` branch on your fork). Otherwise, please comment and we'll take care of them! :)

**Comment by javiereguiluz** [comment]
@seferov thanks a lot for this proposal.

A quick message to our doc mergers: this article is going to be revamped soon. See #7960. I'll finish it next week. So, decide if it's worth it to merge this PR too. Thanks!

**Comment by seferov** [comment]
@wouterj updated as you requested.

@javiereguiluz I just saw your proposal 👍

File: performance.rst [hunk 3]
```diff
@@ -168,6 +168,12 @@ For more details, see :doc:`/components/class_loader/cache_class_loader`.
 Use Bootstrap Files
 -------------------
 
+.. caution::
+
+    The performance optimalizations in PHP 7 removed any overhead. As such,
+    using the bootstrap file is discouraged when using PHP 7+ and was
+    deprecated in Symfony 4.0.
```
Comment (xabbuh) [inline performance.rst]: This was deprecated in 3.3. 4.0 will no longer support the bootstrap files.

**Comment by javiereguiluz** [comment]
Closing it in favor of #9161.

**Pull request closed without merging**